ipn/ipnlocal/serve.go

```diff
@@ -91,7 +92,36 @@ func (s *serveListener) Close() error {
 // Listen is retried until the context is canceled.
 func (s *serveListener) Run() {
 	for {
-		ln, err := net.Listen("tcp", s.ap.String())
+		ip := s.ap.Addr()
+		ipStr := ip.String()
+
+		var lc net.ListenConfig
+		if initListenConfig != nil {
+			// On iOS/macOS, this sets the lc.Control hook to
+			// setsockopt the interface index to bind to, to get
+			// out of the network sandbox.
```

Comment:
can you reword this "to get out of the network sandbox" to explain more what's going on and what the change in behavior is?